packages/bundler-plugin-core/src/sentry/telemetry.ts

```diff
@@ -60,39 +60,30 @@ export function createSentryInstance(
 }
 
 export function setTelemetryDataOnHub(options: NormalizedOptions, hub: Hub, bundler: string) {
-  const {
-    org,
-    project,
-    cleanArtifacts,
-    finalize,
-    setCommits,
-    dryRun,
-    errorHandler,
-    deploy,
-    include,
-    sourcemaps,
-  } = options;
-
-  hub.setTag("include", include.length > 1 ? "multiple-entries" : "single-entry");
+  const { org, project, release, errorHandler, sourcemaps } = options;
+
+  if (release.uploadLegacySourcemaps) {
+    hub.setTag(
+      "uploadLegacySourcemapsEntries",
+      Array.isArray(release.uploadLegacySourcemaps) ? release.uploadLegacySourcemaps.length : 1
+    );
+  }
 
   // Optional release pipeline steps
-  if (cleanArtifacts) {
+  if (release.cleanArtifacts) {
     hub.setTag("clean-artifacts", true);
   }
-  if (setCommits) {
-    hub.setTag("set-commits", setCommits.auto === true ? "auto" : "manual");
+  if (release.setCommits) {
+    hub.setTag("set-commits", release.setCommits.auto === true ? "auto" : "manual");
   }
-  if (finalize) {
+  if (release.finalize) {
     hub.setTag("finalize-release", true);
   }
-  if (deploy) {
+  if (release.deploy) {
     hub.setTag("add-deploy", true);
   }
 
   // Miscelaneous options
```

Comment:
super-l: Should we add a tag for the new deleteAfterUpload option? Might be interesting to see how often it is used?

Reply:
We don't have it implemented yet but it probably doesn't hurt to add right now!